src/flags.cc

```diff
@@ -95,6 +95,7 @@ DEFINE_int32(request_timeout_ms, 20000, "request timeout");
 DEFINE_int32(request_sleep_time, 1000, "the sleep time when request error");
 
 DEFINE_uint32(max_traverse_cnt, 50000, "max traverse iter loop cnt");
+DEFINE_uint32(traverse_cnt, 1000, "traverse iter loop cnt");
```

Comment:
`traverse_cnt` is not a good name? How about `traverse_cnt_limit`?

Reply:
And we'd better to tell the difference between  `max_traverse_cnt ` and `traverse_cnt `